src/polymer-project.ts

```diff
@@ -93,6 +94,10 @@ export class PolymerProject {
     return dependenciesStream;
   }
 
+  addPrefetchLinks(): AddPrefetchLinks {
```

Comment:
Description here, since it's the main user API? Especially the treatment of absolute URLs and checking for the base tag.